### DISCUSSION TITLE
[FEATURE] Python Binding

### DISCUSSION THREAD

**Discussion by edmuthiah** [awslabs/amazon-kinesis-video-streams-producer-sdk-cpp#962] [Ideas]
Please provide Python bindings or wrappers. Most of the ML/CV frameworks lean towards Python so it would make it much easier to modify and produce video streams.

**Comment by disa6302** [comment]
Thank you for the request @ed-muthiah . We will look into this and get back when we have something.

**Comment by edmuthiah** [comment]
@disa6302 Seems like the work has already been done by the DeepLens team?
https://docs.aws.amazon.com/deeplens/latest/dg/deeplens-kinesis-video-streams-api.html

Could the python encapsulation that's being referred to here be made public?

**Comment by edmuthiah** [comment]
@jdelapla If we aren't able to get access to the `DeepLens_Kinesis_Video` library, I'm happy to contribute to this. Please let me know if there's a preferred binding tool PyBind11/SWIG/Boost.Python.
